README.md

```diff
@@ -1,3 +1,33 @@
 Ģ̧̛̳͙̮̞̺̠̯̥͈͈̭͖̦͌̃̃̓̄͂ͬͪ͝l̡̨̠̖͈͉̜͖̘̥͚̫̫̰͕͍̘̗̎̽͌͂̈́͘͞ͅiͬ̋̀̒͏͉̲̺̱̪̲͈͔̼͖̫̗̹̥̠̤͜t̸̺̲̼̬̟̠̗̗̼̙̘̻͕ͭ̃̒ͯ͘͠c̨͉̪̦̥̘̱̜̯͕̪̦̻̼̺͉̖̙̪̫̎̅͗̉̑ͩ̈́ͫ̏̇̍͒̕͡͡h͆̎ͦ͐ͣ͑ͦͬ̍͊ͬ̈̇̀ͩ̀͏̥̣͚̩̹̼̦͚̹̟͔͎̦̣f̈͋̒͡҉̴͈̺̱̣̻͙͝ī̷̞͇̰̟̞̯͕̘ͦ͒̽͐́̏̌ͪ̎̅ͦ͌ͯş̷ͣ͆̇̽̇͏̧̲̖̪̀ͅͅh̵͎͎̭͙̠̺̞̯̗͎͈̪̦̝̼̰̬̯̤ͣͣ̉͛͋̋͛̏͑̇ͫ͋̊̾̚͢͜
 ̡̛̛̳̝̯̣̟̮̳͔̤͕̟̲̫̔̐͋̈̔̾ͧͣ̚
 ͆̒̔̔̏ͣ̏ͣ̽͑̄ͮ̚͏͢҉̨͍̳̞̤̪̘͙̳͙̠͈̟̼̯͔ͅḮ̧̬̤͇̳͍͙͕̬̤̘͉̰͓̮͉̦̜̽̍̍ͯ̇͑ͧͪ̍̀͗̆̅̓̚͢m̷̶̡̡̪͚̳͚̯̙͖̯͍̯̪̗͆̍̃̆͑͋͊ͣ̇̿̔̈͟ảͮ̑͑͆ͪ̀͊̃̑͠҉̵̧̻͚̫̺̗̩͓̰̗̩̬̟͕̦̖̜ͅg̵̛̞͉̭̱̥̰͐̄̂̈̄ͦͯ̊ͯe̟̯͙̝̲̱͖͍͇͙͙̖̭̹̞͔͂̅̂̓ͣ͋̀͘ ̨̡̛̝̤̱̦͇̳̰͎̯̘͍̜̑ͫͮͯͫ̓̋͑͐̿͋̈́ͨ̏͂̈́g̷͖̣͕͈̫̥͉̬̯̘̟̫̳͌ͩ̎̋̉̎̋ͩ̐̑̎ͤ̏̽͊̒̋̕l̵̨̟͎̣̰̙̝̯̖̠͖̭̜̭͑̔̍ͦ͌̆̀̔̒̄̊ͣ̌͌̋̈͌́͜͡i̅̇͊̀́͐͐̄̇ͯ̿̀ͫ̀̚͏̞̜̞̞͖͇͚̘͚̮̞͢͠t̷̨͍͈͚̳̩̺͈̮̗̩̜͙̱̦̂̽̆ͫ̈́͑̔̑̈́ͨͦ̏̏̔̽ͨ̎̽ͧ͢͝ͅç̵̳̬̜̟̖̞͍̖̫̥̹͙͚̍͂͑̿ͅh̖͓͎̖͉̳̱̗͕̱͎̮̹̖͉̲͇ͫ̽̂̊́͢ͅę̫̝̥̝̼͈̠̤̤̰̪̠͇̣̼̱̱̉̈̿̓̂̄͆ͨ̓ͯ͛̂̕ͅr̡̤͉̟̳̭͚͙͚̞̤̳̀͛͒̅͟͠ͅ
+
+
+## How do I deploy this container stack?
+
+See [https://github.com/wetfish/production-manifests](https://github.com/wetfish/production-manifests)
+for production deployment and full stack dev env info.
+
+For development, to run just this stack, do 
+```bash
+cp mariadb.env.example mariadb.env
+# -> edit, change passwords and other info as needed
+cp php.env.example php.env
+# -> edit, change passwords to match
```

Comment:
This project doesn't use mariadb. I'm assuming this was copied from other projects?

Reply:
Yep, and it looks like we don't need that line about php.env either?

Reply:
Yep, there is no local config for this project, it's nearly all client side JS with a bit of PHP to glitch things up

Reply:
Should be resolved by [https://github.com/wetfish/glitch/pull/2/commits/c485280cc32804e73a784ef5aa054a212155abba](https://github.com/wetfish/glitch/pull/2/commits/c485280cc32804e73a784ef5aa054a212155abba)